### PR TITLE
initial model definitions for User, Plant, Order

### LINKED ISSUE #4: Define models (Sequelize)
## In database (Petra)
- [x] seed data of store inventory (plants)
- [x] seed data of users (users)
- [x] seed data of orders with association to users

## Sequelize (Siwin)
- [x] define "plants" model
- [x] define "user" model
- [x] define "orders" model

### PR DESCRIPTION
**Pull request by siwinlo**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [x] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_
* I initialized models for User (very similar to boilermakers), Plant, and Order (closes #4)
* made one-to-many associations for users and orders, and many-to-many for plants and orders

### CODE REVIEW

File: server/db/models/order.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  date: {
+    type: Sequelize.DATE,
+    defaultValue: Date.now()
+  },
+  gift: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
+  checkedOut: {
```
Comment (petradish) [inline server/db/models/order.js]: Do we need this? Should orders refer to purchases (to use in purchase history)?

File: server/db/models/plant.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    allowNull: false
+  },
+  description: {
+    type: Sequelize.TEXT,
+    allowNull: false
+  },
+  price: {
+    type: Sequelize.DECIMAL(10, 2)
+  },
+  quantity: Sequelize.INTEGER
```
Comment (petradish) [inline server/db/models/plant.js]: can you add imgUrl?

File: server/db/models/order.js (hunk 1 above)
Comment (siwinlo) [reply server/db/models/order.js]: I can take it out, since we have a date anyways?

File: server/db/models/plant.js (hunk 2 above)
Comment (siwinlo) [reply server/db/models/plant.js]: ok!

File: server/db/models/order.js (hunk 1 above)
Comment (petradish) [reply server/db/models/order.js]: cool

File: server/db/models/plant.js (hunk 2 above)
Comment (petradish) [reply server/db/models/plant.js]: Also, does order of fields matter? If so, I had my order for Plant as : name, price, quantity, imgUrl, description

**Pull request merged**